spec/glueby/contract/token_spec.rb

```diff
@@ -147,6 +147,21 @@
 
       it { expect { subject }.to raise_error Glueby::Contract::Errors::InsufficientFunds }
     end
+
+    context 'invalid reissuer' do
+      let(:issuer) { wallet }
+      let(:wallet) { TestWallet.new(internal_wallet) }
+      let(:internal_wallet) do
+        class TestInternalWallet < Glueby::Internal::Wallet
+          def get_addresses
+            []
```

Comment:
Here should return addresses list not an empty array. Otherwise, codes in each block wouldn't run on the test.